study-builder/src/main/java/org/broadinstitute/ddp/studybuilder/task/PanCanUniqueCancerValidation.java

```diff
@@ -0,0 +1,59 @@
+package org.broadinstitute.ddp.studybuilder.task;
```

Comment:
We could have combined both patches into one patch and added both validations from one patch.

Reply:
Is it suitable to just combine it in `patch-log.conf`? 
```
{
  "patches": [
    "PanCanOtherCommentsValidation",
    "PanCanUniqueCancerValidation"
  ]
}
```
In such case it will be executed all (when option `--patch` is specified).

Reply:
I meant one java class to handle both but two/separate is ok. 
yes --patch will run all patches in the patch-log.conf file